plugins/python-build/bin/python-build

```diff
@@ -756,7 +756,7 @@ build_package() {
 
   echo "Installing ${package_name}..." >&2
 
-  [ -n "$HAS_PATCH" ] && apply_patch "$package_name" <(cat "${package_name}.patch")
```

Comment:
This is [process substitution](https://www.gnu.org/software/bash/manual/html_node/Process-Substitution.html)